packages/components/src/templates/media.stories.ts

```diff
@@ -0,0 +1,139 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  title: 'Templates/Media',
+  tags: ['!dev'],
+  parameters: {
+    chromatic: { disableSnapshot: true }
+  }
+};
+
+/**
+ * Users can choose more ratios for different needs using the class `aspect-[ratio]`.
+ */
+export const Ratios = {
+  render: () =>
+    html`<div class="grid grid-cols-2 items-end gap-12">
+      <figure class="sd-media aspect-[16/9]">
```

Comment:
I think setting the aspect-ratio in the figure is not needed. Please double check

Reply:
Not needed. Updated